roles/neutronstar/tasks/main.yml

```diff
@@ -39,14 +52,19 @@
   lineinfile:
      dest: "{{ ngi_pipeline_conf }}/{{ bash_env_sthlm_script }}"
      line: >
-         alias neutronstar='nextflow run {{ neutronstar_dest }}/main.nf -profile standard,uppmax
-         -c {{ ngi_pipeline_conf }}/neutronstar_{{ item.site }}.config'
+         alias neutronstar='nextflow run {{ neutronstar_dest }}/main.nf -profile standard,uppmax \
```

Comment:
Does this use the nf-core/configs repo? You should drop`-profile standard,uppmax` for `-profile singularity`

There are still some more options that is more suited and/or needed for irma. As a config file load with `-c`:

```
params {
  busco_folder = '/sw/apps/bioinfo/BUSCO/v2_lineage_sets'
  busco_data = 'eukaryota_odb9'
  clusterOptions = '-A ngi2016003'
  max_memory = 250.GB
  max_cpus = 16
  max_time = 240.h
}
process {
  executor = 'slurm'
}
```

Or alternatively add them as parameters to that alias with `-process.executor='slurm'`

Reply:
I'm a bit unsure if to keep `busco_data = 'eukaryota_odb9'` - I think it's a reasonable default and does make the pipeline more foolproof.

Reply:
I've updated the parameters in this [config file](https://github.com/NationalGenomicsInfrastructure/irma-provision/blob/94b086d5dd978683765f2a8b534c545143359b49/roles/neutronstar/templates/neutronstar_site.config). Also changed the profile to `singularity`.

Reply:
@remiolsen - you should add this config as an uppmax + pipeline specific config in the nf-core/configs repo.

See https://github.com/nf-core/configs/tree/master/conf/pipeline and https://github.com/nf-core/configs/blob/master/pipeline/ and ask @MaxUlysse for help if you need it 👍

Reply:
Note that it's only `params.busco_folder` and `params.busco_data` that looks unique to me. I think everything else is already set up in the uppmax profile.